src/avfoundation.cs

```diff
@@ -3945,6 +4068,43 @@ interface AVAssetWriter {
 		[iOS (8,0), Mac (10,10)]
 		[Export ("directoryForTemporaryFiles", ArgumentSemantic.Copy), NullAllowed]
 		NSUrl DirectoryForTemporaryFiles { get; set; }
+
+		// from category AVAssetWriterSegmentation (AVAssetWriter)
+
+		[TV (14, 0), NoWatch, Mac (11, 0), iOS (14, 0)]
+		[Export ("preferredOutputSegmentInterval", ArgumentSemantic.Assign)]
+		CMTime PreferredOutputSegmentInterval { get; set; }
+
+		[TV (14, 0), NoWatch, Mac (11, 0), iOS (14, 0)]
+		[Export ("initialSegmentStartTime", ArgumentSemantic.Assign)]
+		CMTime InitialSegmentStartTime { get; set; }
+
+		[TV (14, 0), NoWatch, Mac (11, 0), iOS (14, 0)]
+		[Export ("outputFileTypeProfile")]
+		[NullAllowed]
+		[BindAs (typeof (AVFileTypeProfile))]
+		NSString OutputFileTypeProfile { get; set; }
+
+		[Wrap ("WriterWeakDelegate")]
+		IAVAssetWriterDelegate WriterDelegate { get; set; }
+
+		[TV (14, 0), NoWatch, Mac (11, 0), iOS (14, 0)]
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		NSObject WriterWeakDelegate { get; set; }
```

Comment:
Why `Writer*Delegate` instead of `just `*Delegate`? There doesn't seem to be a Delegate property already.

Reply:
This comes from a category. I fear that an other category might be added with a Delegate, which could mean that we will have the Deprecated XAMCORE_4 dance. Since it is a category, it might happen. Unless you see this not being possible, I prefer to add the Writer prefix.

Reply:
Not that I recall, I think we can leave just the `Delegate` name here, it makes sense `AVAssetWriter` is the class name and `AVAssetWriterDelegate` is its delegate.

My vote is to just use `Delegate` and `WeakDelegate` as managed names.